src/qml/filters/fisheye/ui.qml

```diff
@@ -0,0 +1,655 @@
+                ListElement { text: qsTr('Spline 4x4'); value: 0.666 }
+                ListElement { text: qsTr('Spline 6x6'); value: 0.833 }                
+                ListElement { text: qsTr('Lanczos 16x16'); value: 1.0 }                
+            }
+            textRole: 'text'
+            onActivated: {
+                filter.set(qualityParam, qualityModel.get(currentIndex).value)
+            }
+        }
+         Shotcut.UndoButton {
+            onClicked: {
+                filter.set(qualityParam, qualityDefault)
+                qualityCombo.currentIndex = setDefaultIndex(qualityDefault, qualityCombo.count) - 1
+            }
+        }
+
+        // Row 5: Lens Type
+        Label {
+            text: qsTr('Lens type')
+            Shotcut.HoverTip { text: qsTr('Select a lens distortion pattern that best matches your camera') }
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.ComboBox {
+            id: lensCombo
+            implicitWidth: 120
+            model: ListModel { 
+                id: lensModel 
+                ListElement { text: qsTr('Equidistant'); value: 0.0 }
+                ListElement { text: qsTr('Ortographic'); value: 0.333 }
+                ListElement { text: qsTr('Equiarea'); value: 0.666 }
+                ListElement { text: qsTr('Stereographic'); value: 1.0 }
+            }
+            textRole: 'text'
+            onActivated: {
+                filter.set(lensTypeParam, lensModel.get(currentIndex).value)
+            }
+        }
+         Shotcut.UndoButton {
+            onClicked: {
+                filter.set(lensTypeParam, lensTypeDefault)
+                lensCombo.currentIndex = setDefaultIndex(lensTypeDefault, lensCombo.count) - 1
+            }
+        }        
+        
+        //line seperator #2
+        Rectangle{Layout.columnSpan: 3; Layout.fillWidth: true; height: 1; color: activePalette.text; opacity: 0.3}
+
+        // Row 6: Scale (Preset)
+        Label {
+            text: qsTr('Scale (preset)')
+            Shotcut.HoverTip { text: qsTr( 'Preset scale methods\n'+
+                                           'Lock pixels at specific locations') }
+            Layout.alignment: Qt.AlignRight                                           
+        }
+        Shotcut.ComboBox {
+            id: scaleCombo
+            implicitWidth: 120
+            model: ListModel { 
+                id: scaleModel 
+                ListElement { text: qsTr('Scale to Fill'); value: 0.0 }
+                ListElement { text: qsTr('Keep Center Scale'); value: 0.333 }
+                ListElement { text: qsTr('Scale to Fit'); value: 0.666 }
+                ListElement { text: qsTr('Manual Scale'); value: 1.0 }
+            }
+            textRole: 'text'
+            onActivated: {
+                filter.set(scaePresetParam, scaleModel.get(currentIndex).value)
+                scaleShowSlider = ((currentIndex+1)==scaleCombo.count)? true : false //show user input?

```

Comment:
More of these unnecessary `? true : false` things. When I ask for a change please apply it everywhere.

Reply:
i did not realize i could just omit that and it would resolve as bool